_configs/eval_ssm.yaml

```diff
@@ -2,56 +2,82 @@ spreading_model:
   name: "TorchMICModel"
   parameters:
     protocols: ["OR"]
-    p_values: [0.25, 0.5, 0.75]
+    p_values: [0.05, 0.10, 0.15, 0.20]
```

Comment:
(2) this name 'p_values' is confusing, may be thresholds will be more suitable

Reply:
but we're using unfortunately this term everywhere. that's why i decided to keep it here

Reply:
please leave todo for that, I'll change that in free time as low priority task

Reply:
done